### ISSUE TITLE
Wordings change

### ISSUE THREAD

**Issue by Yctxtctx** [simple-login/Simple-Login-Android#104]
Email replied -> Emails sent/replied (Because you can also SEND from alias, not only reply)

![23B89A2E-337B-470C-ADA7-B09AB34CC806](https://user-images.githubusercontent.com/120200229/206858786-05b21df8-508a-4f9d-880a-3a5f9dace1dd.jpeg)

Send Email -> Contacts (because there are contacts, not only “send” feature)

![5BE50829-9F98-4E35-BFD8-A56797D67AD4](https://user-images.githubusercontent.com/120200229/206858838-8eca31be-32e7-4eb1-8ec6-54362421cd22.jpeg)

**Comment by Sherlock1979** [comment]
I agree. Especially the second one (Send email -> Contacts) would be important. I had the impression that the Android app lacks support for Contacts, just because I couldn't find it. Some other users may struggle too and I assume changing the text would not be that hard.

Thanks for your consideration.
